CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ IMPROVEMENTS:
 * resource/nomad_acl_policy: add support for `job_acl` ([#314](https://github.com/hashicorp/terraform-provider-nomad/pull/314))
 * resource/nomad_csi_volume and resource/nomad_csi_volume_registration: add support to import existing volumes. ([#359](https://github.com/hashicorp/terraform-provider-nomad/pull/359)]
 * resource/nomad_job: add support to import existing jobs. ([#359](https://github.com/hashicorp/terraform-provider-nomad/pull/359)]
+* resource/nomad_namespace data source/nomad_namespace: add `node_pool_config` attribute ([#355](https://github.com/hashicorp/terraform-provider-nomad/pull/355))
```

Comment:
Do we want two entries for this, it hurt my brain slight reading it :D

Reply:
Ah good point. I was following the patter of some of the other entries but splitting into two makes sense.